core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -365,7 +365,10 @@ func (i *pluginOracleCreator) createFactoryAndTransmitter(
 				RmnPeerClient:     rmnPeerClient,
 				RmnCrypto:         rmnCrypto})
 		factory = promwrapper.NewReportingPluginFactory[[]byte](factory, i.lggr, chainID, "CCIPCommit")
-		transmitter = plugins[chainFamily].ContractTransmitterFactory.NewCommitTransmitter(destChainWriter,
+		transmitter = plugins[chainFamily].ContractTransmitterFactory.NewCommitTransmitter(
+			i.lggr.Named("CCIPCommitTransmitter").
+				Named(destRelayID.String()),
```

Comment:
I think we can pass in some fields to disambiguate much more, e.g donID, config digest, etc.